pypeit/spectrographs/gemini_gmos.py

```diff
@@ -282,7 +282,7 @@ def get_rawimage(self, raw_file, det):
         # Need the exposure time
         exptime = hdu[self.meta['exptime']['ext']].header[self.meta['exptime']['card']]
         # Return, transposing array back to orient the overscan properly
```

Comment:
I'm confused. GMOS images are multi-extension fits, but you appear to be only passing back extension 0, and extension 1, whereas before the entire hdu was passed back. Not sure this matters, but it is confusing.

Reply:
You're right - we should just use the function `get_headarr()` for all. I'll fix that now for all instruments.